mypy_django_plugin/transformers/managers.py

```diff
@@ -77,8 +167,17 @@ def create_new_dynamic_class(self) -> None:
                 break
             for name, sym in class_mro_info.names.items():
                 if isinstance(sym.node, FuncDef):
+                    # self.copy_method_to_another_class(
```

Comment:
Do we need these lines? Or can we delete them?

Reply:
@sobolevn You're probably right, but at this moment I'm a bit stuck on debugging the existing code (I mean written by @mkurnikov), will fix it later